coverage_comment/coverage.py

```diff
@@ -41,6 +42,8 @@ class FileCoverage:
     missing_lines: list[int]
     excluded_lines: list[int]
     info: CoverageInfo
+    executed_branches: list[list[int]] | None = None
+    missing_branches: list[list[int]] | None = None
```

Comment:
Giving these default values meant moving them _after_ the `info` element, in the `DataClass`. Hopefully that won't cause any problems? The code seems to always use keyword arguments when constructing them.

Reply:
I hate positional arguments when there's more than 1 argument.

Also: the rule that arguments with default must follow positional arguments only holds when arguments may be positional. By indicating to Python that arguments must be named (with `*`), this rule disappears:

```python
def a(b=1, c):  # Syntax error
    ...
def a(*, b=1, c):  # Valid
    ...
a(2, 3)  # TypeError because arguments following `*` MUST be named
```
In this case, I suggest you add a `*` before the first argument, and we'll get a proper exception in case I used positional arguments :)

EDIT: oops, I just noticed it's a dataclass, not a function signature. In that case:
```python
@dataclass(kw_only=True)
```

Reply:
(following my previous comment: let's actually make those mandatory so the problem disappears)
(but you can keep the `kw_only`, it doesn't hurt)